lib/administrate/search.rb

```diff
@@ -88,7 +88,7 @@ def query_template
           attribute_type = attribute_types[attr]
 
           search_target = "#{table_name}.#{column_name}"
-          search_target = "CAST(#{search_target} AS CHAR(256))" unless attribute_type.search_exact?
+          search_target = "CAST(#{search_target} AS CHAR(256))" unless attribute_type.search_skip_cast?
```

Comment:
```suggestion
          search_target = "CAST(#{search_target} AS CHAR(256))" unless attribute_type.search_skip_cast? || attribute_type.search_exact?
```
think we probably still want to keep the old functionality?

Reply:
This change allows each option to control a single change instead of both this and the following change.